src/index.html

```diff
@@ -10,7 +10,45 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+
+  <body class="page">
+    <form
+      class="search"
+      method="get"
+      data-qa="big"
+    >
+      <label
+        class="search__label"
+        for="bigbar">
+      </label>
+      <input
+      class="search__input"
+      type="text"
+      id="bigbar"
+      name="bigbar"
+      data-qa="keypress"
+      placeholder="Try &quot;Los Angeles&quot;"
+      >
```

Comment:
```suggestion
      <label
        class="search__label"
        for="bigbar"
      >
      </label>
      <input
        class="search__input"
        type="text"
        id="bigbar"
        name="bigbar"
        data-qa="keypress"
        placeholder="Try &quot;Los Angeles&quot;"
      >
```
Consider fixing indents according to the code style guide